HotTips/Tips/EDI_OpenRegion.md

```diff
@@ -0,0 +1 @@
+﻿Press **Ctrl+M+O/P** to open and close methods and regions. You can also try **Ctrl+M+H** for current selection.
```

Comment:
Ctrl+M, O will collapse to definitions.
Ctrl+M, P will ... ?? I don't know. What?
Ctrl+M, H will "Hide" the current selection.
Ctrl+M, U will Unhide a hidden block.

But more useful...
Ctrl+M, E will expand a collapsed region
Ctrl+M, S will shrink (collapse) an expanded region.